terraform/modules/relay-app/templates/cloud-init.yaml

```diff
@@ -13,7 +13,7 @@ write_files:
         otlp:
           protocols:
             grpc:
-            http:
+              endpoint: localhost:4317
```

Comment:
Please apply the same change to cloud-init of the Elixir app. We also only the HTTP protocol there and grpc can be disabled.

Reply:
Done. From memory, the default HTTP port is `4318` right?

Reply:
Double checked, and found it in the terraform config!